_includes/fancyboxtemplate.html

```diff
@@ -1,4 +1,5 @@
-{{ include.gallery }}
+Gallery: {{ include.gallery }}
+Thumbnail URL: {{ site.galleries[include.gallery].images[0].thumbnail }}
 {% for image in site.galleries[include.gallery].images %}
```

Comment:
On a side note, my eventual implementation didn't need this, it only needed to use `page.galleries` instead of `site.galleries`.